src/widgets/WidgetMainQmc.vue

```diff
@@ -97,6 +98,14 @@ export default defineComponent({
       isDialogOpen: false,
       buttonCloseLabel: 'Close',
       Theme: ThemeEnum,
+      primaryProduct: {
+        productType: '' as ProductEnum,
+        productItems: [] as TabItemProps[],
+      },
+      cardAccountFee: undefined as number | undefined,
+      cardStandardInterestRate: undefined as number | undefined,
```

Comment:
What's happening here?

Reply:
sorted. thanks for the idea @larsklavenes 
![image](https://user-images.githubusercontent.com/63985172/140664724-dd735b87-cc07-4754-9bbc-66ba0f52fce1.png)